Refit.Benchmarks/StaticFileHttpResponseHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Refit.Benchmarks
+{
+    public class StaticFileHttpResponseHandler : HttpMessageHandler
+    {
+        private readonly HttpStatusCode responseCode;
+        private readonly string responsePayload;
+
+        public StaticFileHttpResponseHandler(string fileName, HttpStatusCode responseCode)
+        {
+            if (string.IsNullOrEmpty(fileName))
+                throw new ArgumentNullException(nameof(fileName));
+
+            responsePayload = File.ReadAllText(fileName);
```

Comment:
Should this handler be reading the file from disk each time? Or should we preload the files into memory during the setup stage and returned cached bytes?

That would prevent the tests from being affected by disk I/O.

Reply:
This runs as part of [GlobalSetup], and this disk access happens in the constructor, so it runs only a single time, and isn't having to hit the disk and do I/O on each iteration during the benchmark. During the benchmarks it's just taking the string which it's got in memory and returning it.